frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/sidebar_top_menu.dart

```diff
@@ -49,9 +49,11 @@ class SidebarTopMenu extends StatelessWidget {
     final name = Theme.of(context).brightness == Brightness.dark
         ? FlowySvgs.flowy_logo_dark_mode_xl
         : FlowySvgs.flowy_logo_text_xl;
+
     return FlowySvg(
       name,
       size: const Size(92, 17),
+      blendMode: null,
```

Comment:
![image](https://github.com/AppFlowy-IO/AppFlowy/assets/44445638/5d13e49d-f63a-4af8-aace-bfbd1b3e9572)

Reply:
The variable name doesn't make sense here. Let me fix that.

Reply:
816d01cef389002040ed641c9a0fecd8ab61781e